internal/criocli/status.go

```diff
@@ -159,3 +175,38 @@ func goroutines(c *cli.Context) error {
 
 	return nil
 }
+
+func heap(c *cli.Context) error {
+	crioClient, err := crioClient(c)
+	if err != nil {
+		return err
+	}
+
+	data, err := crioClient.HeapInfo(c.Context)
+	if err != nil {
+		return err
+	}
+
+	outputPath := c.String("file")
+	if outputPath == "" {
+		outputPath = "heap.out"
+	}
+
+	var file *os.File
+
+	if outputPath == "-" {
+		file = os.Stdout
+	} else {
```

Comment:
I wonder if this `outputPath` setting could be a select? We repeat some of the if-else statements checks.

Reply:
Yes, cleaned that up.